docs/content/preview/explore/connection-manager/connection-mgr-ysql.md

```diff
@@ -28,7 +28,7 @@ YugabyteDB includes a built-in connection pooler, YSQL Connection Manager, which
 YSQL Connection Manager is currently not supported for [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Managed](../../../yugabyte-cloud/).
 {{< /note >}}
 
-<!-- ----- Add Diagram---- -->
+![Connection manager](/images/explore/connection-manager.png)
 
 ## Key features
 
```

Comment:
below says
"set the yb-tserver flag enable_ysql_conn_mgr flag to true."
drop a 'flag', ie "set the yb-tserver flag enable_ysql_conn_mgr to true."